app/src/main/java/com/essycynthia/moviechat/ui/login_screens/LoginScreen.kt

```diff
@@ -73,8 +75,10 @@ import com.essycynthia.moviechat.R
 fun LoginScreen(
     navigateToForgot: ()-> Unit,
     navigateToSignUp:()-> Unit,
-    navigateToChat:()-> Unit
+    navigateToChat:()-> Unit,
+    loginScreenViewModel: LoginScreenViewModel = hiltViewModel()
 ) {
+    val uiState by loginScreenViewModel.uiState.collectAsState()
```

Comment:
Please correct your line to this
val uiState = loginScreenViewModel.uiState.collectAsState()